packages/data-stores/src/add-ons/hooks/use-available-storage-add-ons.ts

```diff
@@ -16,9 +16,9 @@ interface Props {
 export function isStorageQuantityAvailable( quantity: number, storage: SiteMediaStorage ): boolean {
 	const existingAddOnStorage = storage.maxStorageBytesFromAddOns / Math.pow( 1024, 3 );
 	const currentMaxStorage = storage.maxStorageBytes / Math.pow( 1024, 3 );
-	const availableStorageUpgrade = STORAGE_LIMIT - currentMaxStorage;
+	const isWithinStorageLimit = STORAGE_LIMIT >= currentMaxStorage - existingAddOnStorage + quantity;
 
-	return existingAddOnStorage < quantity && quantity <= availableStorageUpgrade;
+	return existingAddOnStorage < quantity && isWithinStorageLimit;
```

Comment:
I missed this in my first pass - is there a way to test this change?

Rewriting the existing code, it should be:
```js
const isWithinStorageLimit = STORAGE_LIMIT >= currentMaxStorage + quantity;
```

I do not understand why we subtract `existingAddOnStorage` here. 🤔